tests/covalent_dispatcher_tests/_service/app_test.py

```diff
@@ -109,7 +173,16 @@ def __init__(self, dbpath=dbpath):
     assert result == dbpath
 
 
-def test_get_result(mocker, app, client, test_db_file, tmp_path):
+def test_db_path_exception(mocker, client):
```

Comment:
let's remove as we will remove the db-path route entirely

Reply:
Good point! Removed